profiles/templates/components/education-form.html

```diff
@@ -0,0 +1,80 @@
+          class="w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-lime-300" />
+      </div>
+      <label for="id_current_education" class="ms-2 text-sm font-medium text-gray-900">I am currently
+        studying here</label>
+    </div>
+    <div class="md:w-[50%]">
+      {{ education_form.grade|as_crispy_field }}
+    </div>
+    <div class="hidden" id="education-modules" name="modules" data-item-id="0"></div>
+    <div class="hidden" id="education-skills" name="skills" data-item-id="0"></div>
+  </form>
+  <div class="md:w-[50%]">
+    <!-- Education Modules -->
+    <form data-main-input="education-modules" id="module-form" class="add-item-form mb-6">
+      {% csrf_token %}
+      <label for="new-module-text" class="block text-gray-700 text-sm font-bold mb-2">Modules Covered:</label>
+
+      <ul id="module-list" class="item-list pl-6 mb-3"></ul>
+
+      <div class="flex">
+        <div class="relative w-full">
+          <input type="text" id="new-module-text" name="module"
+            class="add-item-input block p-2.5 w-full z-20 text-sm text-gray-900 bg-gray-50 rounded-l-lg rounded-e-lg rounded-s-gray-100 rounded-s-2 border border-gray-300 focus:ring-lime-300 focus:border-lime-500"
+            placeholder="Module" maxlength="150" required />
+          <button type="submit" id="submit-module"
+            class="add-item-submit absolute top-0 end-0 px-5 py-2.5 h-full text-sm font-medium text-black bg-lime-500 rounded-e-lg border border-lime-600 hover:not(disabled):bg-lime-400 focus:ring-4 focus:outline-none focus:ring-lime-300">
+            Add</button>
+        </div>
+      </div>
+    </form>
+    <!-- Education skills -->
+    <form data-main-input="education-skills" id="education-skill-form" class="add-item-form mb-6">
+      {% csrf_token %}
+      <label for="education-skill-text" class="block text-gray-700 text-sm font-bold mb-2">Applied Skills:</label>
+      <ul id="education-skill-list" class="item-list pl-6 mb-3"></ul>
+
+      <div class="flex">
+        <div class="relative w-full">
+          <input type="text" id="education-skill-text" name="education-skill"
+            class="add-item-input block p-2.5 w-full z-20 text-sm text-gray-900 bg-gray-50 rounded-l-lg rounded-e-lg rounded-s-gray-100 rounded-s-2 border border-gray-300 focus:ring-lime-300 focus:border-lime-500"
+            placeholder="Skill" maxlength="30" required />
+          <button type="submit" id="submit-education-skill"
+            class="add-item-submit absolute top-0 end-0 px-5 py-2.5 h-full text-sm font-medium text-black bg-lime-500 rounded-e-lg border border-lime-600 hover:not(disabled):bg-lime-400 focus:ring-4 focus:outline-none focus:ring-lime-300">
+            Add</button>
+        </div>
+      </div>
+    </form>
+  </div>
+  <button type="submit" form="education-form"
+    class="text-black rounded-lg bg-lime-500 py-2 px-3 border border-lime-600">Add Work
+    Experience</button>
```

Comment:
Issue (Blocking): This should be "Add education"

Reply:
Fixed